app/views/widgets/home_page/_hero_panel.html

```diff
@@ -8,7 +8,7 @@
   </div>
   <section id="hero" class="dark-bg align-left" style="background-image:url('http://www.chapman.edu/_files/img/hero-panels/fallback-images/hilbert-art.jpg');">
     <div id="mastheadContent" class="maxWidth">
-      <span class="tagline" role="heading">Optional Tagline goes here</span>
+      <span class="tagline" role="heading">Optional Tagline goes here <span class="tagline_line-2">Optional Tagline, Line 2</span></span>
```

Comment:
When we make changes to formats in Cascade, we've started adding them to the GitHub repository, under the `.cascade-code/` directory, in the folder that corresponds to the location within Cascade. It's a nonfunctional folder obviously, you can't do any live testing but we do it to allow code reviews in pull requests. So can you please add the Hero Panel Preview format to the right folder in `.cascade-code/` along with any other formats or blocks you changed?

I failed to mention it in the last story you worked on for the Home page Hero, it really should have been added in that story. Please be aware of this practice for any future work whenever you make changes to blocks or formats in Cascade.

See this page of our best practices for more details: https://wimops.chapman.edu/wiki/Cascade_Best_Practices